extras/sdio/sdio.c

```diff
@@ -141,6 +141,8 @@ static uint16_t crc_ccitt(const uint8_t *data, size_t n)
 #define spi_skip_word()   do { spi_read_byte(); spi_read_byte(); } while(0)
 #define spi_skip_dword()  do { spi_read_byte(); spi_read_byte(); spi_read_byte(); spi_read_byte(); } while(0)
 
+#define timeout_expired(start, len) ((uint32_t)(sdk_system_get_time() - (start)) >= (len))
```

Comment:
No, it's not because of the unsigned math:
```C
uint32_t diff = (uint32_t)10000 - (uint32_t)0xfffffff0;
printf("Difference: %d\n", diff);
```
gives
```
Difference: 10016
```